### ISSUE TITLE
Add screenshot to the README.md

### ISSUE THREAD

**Issue by Themercee** [Netflix/Scumblr#237]
This is more an advice. You should add some screenshot to let people know there is a UI. People loooooove nice thing 😍

**Comment by sbehrens** [comment]
Hi @Themercee,

We have a ton of screenshots on our Wiki, where you can fine our complete documentation.  Take a look here to get started: https://github.com/Netflix/Scumblr/wiki/Setup-Your-First-Security-Task

I'll throw a few more screenshots in the readme as well.

**Comment by Themercee** [comment]
Yeah I look at the Wiki, but before looking I didn't know it was Web UI. I thought it was only command line on something like that! Nice work by the way
